src/App.tsx

```diff
@@ -57,22 +57,20 @@ export class App extends React.Component<{}, State> {
           <tbody>
             {goods.map((good) => {
               const isSelected = selectedGood === good;
-              const buttonClassName = `button ${
-                isSelected ? 'has-background-success-light' : ''
-              }`;
-              const minusStyle = isSelected ? { color: 'white' } : {};
+              const trClassName = isSelected
+                ? 'has-background-success-light' : '';
+              const buttonClassName = `button ${isSelected ? 'is-info' : ''}`;
```

Comment:
just FYI - it's better to use libraries as `classnames` for conditional class names